src/controllers/request.controller.js

```diff
@@ -0,0 +1,25 @@
+import RequestService from '../services/request.service';
+import ResponseService from '../services/response.service';
+
+/**
+ *
+ *
+ * @class RequestController
+ */
+class RequestController {
+  /**
+   *
+   *
+   * @static
+   * @param {req} req
+   * @param {res} res
+   * @returns {response} @memberof RequestController
+   */
+  static async rejectRequest(req, res) {
```

Comment:
You can make this method flexible by getting the request status from the request payload then set it here. This method can then be used to approve or reject a request.